internal/sourcecode/templates/laravel/octane/docker/supervisor.conf

```diff
@@ -0,0 +1,63 @@
+[supervisord]
+user=root
+nodaemon=true
+loglevel=info
+logfile=/dev/stdout
+pidfile=/var/run/supervisord.pid
+logfile_maxbytes=0
+
+[group:laravel-worker]
+priority=999
+;programs=nginx,octane,laravel-schedule,laravel-notification,laravel-queue
+programs=nginx,octane
+
+[program:nginx]
+priority=10
+autostart=true
+autorestart=true
+stdout_events_enabled=true
+stderr_events_enabled=true
+command=nginx -g 'daemon off;'
+stdout_logfile=/dev/stdout
+stdout_logfile_maxbytes=0
+stderr_logfile=/dev/stderr
+stderr_logfile_maxbytes=0
+
+[program:octane]
+user=app
+priority=5
+autostart=true
+autorestart=true
+stdout_events_enabled=true
+stderr_events_enabled=true
+;rr command=php -d variables_order=EGPCS /var/www/html/artisan octane:start --server=roadrunner --host=0.0.0.0 --rpc-port=6001 --port=8000
```

Comment:
What is the purpose of these commented-out lines?

Reply:
The `ENTRYPOINT` script uncomments the correct line depending on if they use Octane's `roadrunner` or `swoole` drivers

For added context: Octane is a way to run PHP apps more like other apps - PHP is a long-running process that can accept web requests. 

Traditional PHP spins up a new process and reloads the entire framework for every web request.